codegen/src/test/resources/software/amazon/awssdk/codegen/poet/auth/scheme/query-endpoint-auth-params-auth-scheme-endpoint-provider-without-allowlist.java

```diff
@@ -68,15 +70,13 @@ public List<AuthSchemeOption> resolveAuthScheme(QueryAuthSchemeParams params) {
                                                                               "Expecting auth scheme of class SigV4AuthScheme, got instead object of class %s", authScheme.getClass()
                                                                                                                                                                           .getName());
                     List<String> signingRegionSet = sigv4aAuthScheme.signingRegionSet();
-                    if (signingRegionSet.size() == 0) {
+                    RegionSet regionSet = RegionSet.create(signingRegionSet.stream().collect(Collectors.joining(",")));
+                    if (regionSet.toString().isEmpty()) {
```

Comment:
odd too see isEmpty check here, I assume we wouldn't allow creating RegionSet with empty string.

Reply:
turns out you do not allow it in the constructor, so this can be removed.